react-components/src/apps/admin/AllStats/AppStatsComponents/AppsTab.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * This function contains the AppsTableHeader and the NumberSelect box.
+ * AppsTable contains the actual table which is imported here.
+ */
+
+import React from "react";
+import Box from "@material-ui/core/Box";
+import AppsTable from "./AppsTable";
+import NumberTextfield from "./NumberTextfield";
+import styles from "../AllStatsStyle";
+import { withStyles } from "@material-ui/core";
+import ids from "./AllStatsIDs";
+import messages from "./messages";
+import { withI18N, getMessage, build } from "@cyverse-de/ui-lib";
+
+function AppTab(props) {
+    const { classes } = props;
+    const { baseId } = props.id;
```

Comment:
This [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) doesn't look quite right. I think the `props` object looks something like `{classes: {...}, id: "..."}`, in which case these 2 lines could be combined to destructure both `classes` and `id` at once:
```javascript
     const { classes, id } = props;
```
And if you really want to name the variable `baseId`, then:
```javascript
     const { classes, id: baseId } = props;
```